benchmark/websocket/simple.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
```

Comment:
Can you add `test/benchmark/test-benchmark-websocket.js` test file?

Reply:
Sure, I'll add that test file. Thanks for the suggestion.

Reply:
I added the `test/benchmark/test-benchmark-websocket.js` file.